nmpolicy/internal/parser/parser_test.go

```diff
@@ -36,6 +36,12 @@ func TestParser(t *testing.T) {
 
 func testParseFailures(t *testing.T) {
 	var tests = []test{
+		{
+			tokens: []lexer.Token{},
+			expected: expected{
+				ast: &ast.Node{},
+			},
+		},
```

Comment:
Don't you need to give a description to what is tested here?
And I would wrap it with a `expectSuccess`, similar to what you did in the errors ones.

Reply:
Description is composed from the expression, https://github.com/nmstate/nmpolicy/blob/main/nmpolicy/internal/parser/parser_test.go#L277-L282.

Adding a expectSuccess with empty stuff.